manuscript.md

```diff
@@ -407,6 +409,7 @@ maps seem to agree with species interaction data. However, the disagreement
 between the IUCN and the GBIF databases is concerning and suggests that the IUCN
```

Comment:
Watch out, the 50.7% refers to the GBIF range, not the IUCN one. This should instead read:
> but 50.7% of its GBIF pixels did not superpose with the IUCN range .

Reply:
I would also avoid referring to the 26.4% statistics on the GBIF point data. It's too confusing to have both a value for the pixels and the points. Only the pixels statistic value should be enough

Reply:
Fixed on [f5a7f74](https://github.com/graciellehigino/ms_range_interactions/pull/98/commits/f5a7f74ebdcdc3c66f8a27826bbcf7e057737346)